backends/azure_table_test.go

```diff
@@ -1,14 +1,21 @@
 package backends
 
 import (
-	"github.com/satori/go.uuid"
+	"github.com/gofrs/uuid"
 	"testing"
 )
 
+var u1 = uuid.Must(uuid.NewV4())
```

Comment:
Can I ask what's the purpose of `u1`? Both, here and in the other file. I don't see it being used anywhere.

Reply:
According to [https://github.com/gofrs/uuid](https://github.com/gofrs/uuid) it is used to initialize package-level variables

Reply:
Mansi I just removed that line and I found no difference in the test cases when running `./validate.sh` I believe I'll remove as you suggested